packages/events/defaultSubscribers/migrate/Messages.js

```diff
@@ -275,7 +277,9 @@ class Messages {
 
       // Transactions
       endTransaction: () => {
-        if (reporter.blockSpinner) reporter.blockSpinner.stop();
+        if (spinners.pick("migration-reporter")) {
+          spinners.remove("migration-reporter");
+        }
```

Comment:
Thoughts on moving this guard to the `spinners.remove` function?

Reply:
good call!